code/game/objects/structures/stool_bed_chair_nest/xeno_nest.dm

```diff
@@ -332,7 +332,7 @@
 
 /obj/structure/bed/nest/attack_animal(mob/living/M as mob)
 	M.visible_message(SPAN_DANGER("\The [M] tears at \the [src]!"), \
-		SPAN_DANGER("You tear at \the [src]."))
+		SPAN_DANGER("We tear at \the [src]."))
```

Comment:
Don't change attack_animal, these aren't only xenos (and players would play real xenos anyway)